[New KB article] Collect Semgrep GitHub Actions logs (#1086)
returntocorp/semgrep-docs: docs/kb/semgrep-ci/collect-gha-logs.md

# Collecting Semgrep GitHub Actions logs from Github

1. On Github.com, or in your GitHub instance, navigate to the main page of the repository that Semgrep scanned.
2. Click the **Actions** tab.      
    ![actions-tab](https://docs.github.com/assets/cb-45938/images/help/repository/actions-tab.png)  

> 3. In the left sidebar, click the GitHub Actions workflow you want to retrieve logs for. The name depends on your configuration but defaults to "Semgrep". In this example it is "Scan":

Reviewer: ```suggestion
3. In the left sidebar, click the GitHub Actions workflow you want to retrieve logs for. The name depends on your configuration but defaults to "Semgrep." In this example it is "Scan":
```
[GSG thing: Commas and periods go inside quotation marks](https://developers.google.com/style/quotation-marks#commas-and-periods-with-quotation-marks)

    ![GitHub actions workflow selection](/img/kb/semgrep-gha-workflow.png)
4. Select the desired workflow run.
5. Click on the Semgrep job name. The name depends on your configuration but defaults to "semgrep/ci":
    ![Semgrep job name](/img/kb/semgrep-gha-jobname.png)

> 6. In the upper right corner of the log, select the settings gear, then click **Download log archive**.

Reviewer: ```suggestion
6. In the upper right corner of the log, click the <i class="fa-solid fa-gear"></i> dropdown menu > **Download log archive**.
```
Purely a style thing. I chose to go with how GitHub writes the line except for the > which are a GSG thing ([small actions](https://developers.google.com/style/procedures#multi-action-procedures))

Reviewer: Btw, I have a small library of snippets in my vim (easily portable to other editors) I think, that fill the gap for easy editing, it helps me easily write tables, lists, and those sick fontawesome icons. Let me know if you're interested, I've been holding off making a repo because it's not yet that extensive.
<img width="654" alt="Snag_165cb27" src="https://github.com/returntocorp/semgrep-docs/assets/20766840/f3b52a4f-9f6f-4fa7-811d-853c270e426a">

Author: Oh yes, that would be amazing! I didn't know we had one, although now that I think of it, I've seen it in other places.

7. Share the downloaded log archive with Semgrep Support.

> [See these instructions in GitHub Docs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/using-workflow-run-logs#downloading-logs).

Reviewer: ```suggestion

## Additional references

* [Using workflow run logs - Downloading logs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/using-workflow-run-logs#downloading-logs).
```

Reviewer: We use an "Additional references" header for external links we think people are interested in! Such as: [SSC overview](https://semgrep.dev/docs/semgrep-supply-chain/overview/#additional-references)
